docs/networking/best-practice-vlan-aware-switch.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 50
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Network
+  - network
+  - VLAN
+  - vlan
+Description: Harvester is built on top of Kubernetes, and uses the [CNI](https://github.com/containernetworking/cni) as the interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, the Harvester UI integrates the network configuration in order to provide a user-friendly way to configure networks for VMs.
+---
+
+# Best Practice for VLAN-aware Switch
+
+In this best practice guide on how to configure "VLAN-aware", we will introduce Harvester VLAN network and external switch configuration for common scenario.
+
+## Architecture
+
+Hardware:
+- Three Harvester servers with only one single port network card.
+- One or more VLAN-aware switch(es). We will use "Cisco like" configuration as example.
+
+Network Specification:
+- The Management Network to control Harvester servers needs to be set as VLAN 100.
+- The VM Network needs to be in the VLAN 101-200 range.
```

Comment:
Suggestion to change this sentence as:
Assume that the VMs are in the VLAN 101-200.